src/test/scala/nl/knaw/dans/easy/stage/RunSpec.scala

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage
+
+import java.io.File
+import java.nio.file.{Files, Path, Paths}
+
+import org.apache.commons.io.FileUtils
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.util.Success
+
+class RunSpec extends FlatSpec with Matchers {
+  private val testDir = Paths.get("target/test", getClass.getSimpleName)
+  FileUtils.deleteQuietly(testDir.toFile)
+  Files.createDirectories(testDir)
+
+  /*
+   * Problems with this test:
+   *
+   * - It depends on what is in src/test/resources/dataset-bags. If one of the bags changes or bags are added
+   *   or removed, this test may unexpectedly break.
+   * - It runs a very high-level function (almost `main` itself) and there depends on the
+   *   complete application context (settings, licenses, etc).
```

Comment:
The advantage is the test resources won't get a change to decay. But solid validation and tests for the bags and configuration (like for the deposits / application.properties in ingest-flow) could cater for that.